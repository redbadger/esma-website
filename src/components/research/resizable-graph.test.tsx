import React from "react";
import renderer from "react-test-renderer";
import ResizableGraph from "./resizable-graph";
import BlocksSVG from "../../svg/blocks.svg";
import BookSVG from "../../svg/book.svg";
import CapSVG from "../../svg/cap.svg";

describe("ResizableGraph", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <ResizableGraph mobile={CapSVG} desktop={BookSVG} tablet={BlocksSVG}>
          <p>Sample heading</p>
        </ResizableGraph>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
