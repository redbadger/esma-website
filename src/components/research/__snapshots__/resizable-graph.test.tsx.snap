// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResizableGraph renders correctly 1`] = `
<div
  className=" css-i5zxf-ResizableGraph"
>
  <p>
    Sample heading
  </p>
  <test-file-stub />
</div>
`;
